cloud/user/user.go

```diff
@@ -0,0 +1,49 @@
+package user
+
+import (
+	"fmt"
+
+	"github.com/astronomer/astro-cli/astro-client"
+	"github.com/astronomer/astro-cli/software/workspace"
+	"github.com/pkg/errors"
+)
+
+// CreateInvite calls the CreateUserInvite mutation to create a user invite
+func CreateInvite(email, role string, client astro.Client) (astro.UserInvite, error) {
+	var userInviteInput astro.CreateUserInviteInput
+	userInviteInput.InviteeEmail = email
+	userInviteInput.Role = role
+	derivedOrganizationID, err := getOrganizationID(client)
+	if err != nil {
+		return astro.UserInvite{}, err
+	}
+	userInviteInput.OrganizationID = derivedOrganizationID
+	return client.CreateUserInvite(userInviteInput)
```

Comment:
```suggestion
	derivedOrganizationID, err := getOrganizationID(client)
	if err != nil {
		return astro.UserInvite{}, err
	}
	userInviteInput := astro.CreateUserInviteInput{InviteeEmail: email, Role: role, OrganizationID: derivedOrganizationID}
	return client.CreateUserInvite(userInviteInput)
```
nitpick: a slightly cleaner way to do the same thing, up to you on how you want to go by

Reply:
like it and will refactor to your suggestion.